target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-ad7200.dts

```diff
@@ -0,0 +1,453 @@
+#include "qcom-ipq8064-v1.0.dtsi"
+#include <dt-bindings/input/input.h>
```

Comment:
add a blank line before the dt-bindings and order the include alphabetical (gpio before input)